src/dc_receive_imf.rs

```diff
@@ -1140,15 +1140,7 @@ async fn create_or_lookup_group(
             member_ids.push(DC_CONTACT_ID_SELF);
         }
 
-        // Only search for parent message if there are more than two
-        // members to make sure private replies to group
-        // messages are assigned to a 1:1 chat.
-        //
-        // If group contains only 2 members, there is no way to
-        // distinguish between "Reply All" and "Reply", so replies
-        // to such groups from classic MUAs will be always assigned to
-        // 1:1 chat.
-        if member_ids.len() > 2 {
+        if mime_parser.get(HeaderDef::ChatVersion).is_none() {
```

Comment:
maybe add a comment as "We do not check the parent-message to avoid private replies sorted to the wrong chats (private replies use the same In-Reply-To and References header to make references between quotes work and to allow non-delta-MUAs to sort them to the correct threads)"